cluster/gce/gci/configure-helper.sh

```diff
@@ -2051,6 +2051,33 @@ function setup-coredns-manifest {
   sed -i -e "s@{{ *pillar\['service_cluster_ip_range'\] *}}@${SERVICE_CLUSTER_IP_RANGE}@g" "${coredns_file}"
 }
 
+# Sets up the manifests of Fluentd configmap and yamls for k8s addons.
+function setup-fluentd {
+  local -r dst_dir="$1"
+  local -r fluentd_gcp_yaml="${dst_dir}/fluentd-gcp/fluentd-gcp-ds.yaml"
+  # Ingest logs against new resources like "k8s_container" and "k8s_node" if
+  # LOGGING_STACKDRIVER_RESOURCES is "new".
+  # Ingest logs against old resources like "gke_container" and "gce_instance" if
+  # LOGGING_STACKDRIVER_RESOURCES is "old".
+  if [[ "${LOGGING_STACKDRIVER_RESOURCES:-}" == "new" ]]; then
```

Comment:
I confused RESOURCES with actual resources as defined by Kubernetes, this is a nitpick on my part, but I'd petition to change this to `LOGGING_STACKDRIVER_RESOURCE_TYPES`, or anything else that doesn't blur the line of cpu / memory resources. Even resource types might be confusing. This could all just be unnecessary noise, ignore me if that keeps things rolling :)

https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/

Reply:
That's a valid point. Renamed to `LOGGING_STACKDRIVER_RESOURCE_TYPES`. It's not ideal, but at least less confusing.